test/functional/rpc_rawtransaction.py

```
@@ -314,8 +314,8 @@ def run_test(self):
         # 5. valid parameters - supply txid and True for non-verbose
         assert_equal(self.nodes[0].getrawtransaction(txHash, True)["hex"], rawTxSigned['hex'])
 
-        # 6. invalid parameters - supply txid and string "Flase"
-        assert_raises_rpc_error(-1, "not a boolean", self.nodes[0].getrawtransaction, txHash, "Flase")
+        # 6. invalid parameters - supply txid and string "False"
+        assert_raises_rpc_error(-1, "not a boolean", self.nodes[0].getrawtransaction, txHash, "False")
```

Ruff (0.15.7)

[error] `assert_raises_rpc_error` may be undefined, or defined from star imports

(F405)